doc/install_generic.inc

```diff
@@ -121,3 +125,8 @@ libgpuarray
 
 Install the latest, development version of libgpuarray following the
 `Step-by-step instructions <http://deeplearning.net/software/libgpuarray/installation.html#step-by-step-install>`__.
+
+Alternatively, we also provide a conda channel ``mila-udem`` which contains packages for ``pygpu`` and ``libgpuarray``
```

Comment:
Remove the new stuff and write something like:

See note in the bleeding edge version about pygpu 0.7.1